### ISSUE TITLE
Can i use it for this?

### ISSUE THREAD

**Issue by 9acca9** [boochow/micropython-raspberrypi#36]
So... What is this? I have a two python files I need to run at the boot on a RPI zero w. Those files call i2c components. A multiplexor, 4 mcp, 1 htk1633 for LEDs and a LCD display. (And I need wifi)

Can I use micro python to make that work? (im currently using raspbian).

**Comment by boochow** [comment]
No, I'm sorry, this does not support WiFi. I recommend ESP32 if you want to use WiFi on micropython.
